src/Agent.Worker/ContainerOperationProvider.cs

```diff
@@ -452,5 +454,101 @@ private async Task<List<string>> ExecuteCommandAsync(IExecutionContext context,
             return outputs;
         }
 #endif
+
+        public async Task CreateContainerNetworkAsync(IExecutionContext executionContext, object data)
+        {
+            Trace.Entering();
+            ArgUtil.NotNull(executionContext, nameof(executionContext));
+            string containerNetwork = data as string;
+            int networkExitCode = await _dockerManger.DockerNetworkCreate(executionContext, containerNetwork);
+            if (networkExitCode != 0)
+            {
+                throw new InvalidOperationException($"Docker network create failed with exit code {networkExitCode}");
+            }
+
+            // Expose docker network to env
+            executionContext.Variables.Set(Constants.Variables.Agent.ContainerNetwork, containerNetwork);
+        }
+
+        public async Task RemoveContainerNetworkAsync(IExecutionContext executionContext, object data)
+        {
+            Trace.Entering();
+            ArgUtil.NotNull(executionContext, nameof(executionContext));
+            string containerNetwork = data as string;
+            if (string.IsNullOrEmpty(containerNetwork))
+            {
+                containerNetwork = executionContext.Variables.Get(Constants.Variables.Agent.ContainerNetwork);
+            }
+            if (!string.IsNullOrEmpty(containerNetwork))
+            {
+                executionContext.Output($"Remove container network: {containerNetwork}");
+
+                int removeExitCode = await _dockerManger.DockerNetworkRemove(executionContext, containerNetwork);
+                if (removeExitCode != 0)
+                {
+                    executionContext.Error($"Docker network rm failed with exit code {removeExitCode}");
+                }
+                // Remove docker network from env
+                executionContext.Variables.Set(Constants.Variables.Agent.ContainerNetwork, null);
+            }
+        }
+
+        public async Task StartMultipleContainersAsync(IExecutionContext executionContext, object data)
```

Comment:
move public method up and move private method down in the file.

Reply:
StartContainersAsync?

Reply:
Done